assets/src/application/ui/forms/espressoForm/styles.scss

```diff
@@ -75,69 +101,137 @@
 	.section-body {
 		padding: var(--ee-padding-smaller) 0;
 
+		@include max782px {
+			flex: 0 100%;
+		}
+
+		@include min782px {
+			flex: 0 80%;
+			position: relative;
+			.field-group-item {
+				flex: 0 50%;
+				padding: 0 0.5em;
+			}
+		}
+
 		.ee-form-item {
 			padding: var(--ee-padding-smaller) 0;
 
+			@include max782px {
+				padding: var(--ee-padding-tiny) 0;
+			}
+
+			@include max600px {
+				padding: var(--ee-padding-micro) 0;
+			}
+
+			@include max480px {
+				padding: var(--ee-padding-nano) 0;
+			}
+
 			&-switch {
 				padding: var(--ee-padding-small) 0;
 			}
 
 			&-pair {
-				width: 50%;
 				display: inline-flex;
-				padding-right: var(--ee-margin-default);
 				margin: var(--ee-margin-default) 0;
-				label {
-					display: flex;
+				padding-right: var(--ee-padding-big);
+				width: 50%;
+
+				label:first-child {
 					align-items: center;
+					display: flex;
+					padding: var(--ee-padding-nano) var(--ee-padding-smaller) 0 0;
+					min-width: 200px;
 				}
-			}
 
-			.ee-icon--info-circle-outlined {
-				color: var(--ee-color-primary-low-contrast);
-				margin: 0 var(--ee-margin-tiny);
+				> div {
+					width: calc(50% - 200px);
+				}
+
+				@include max782px {
+					margin: var(--ee-margin-small) 0;
+					width: 100%;
+
+					label:first-child {
+						justify-content: flex-end;
+						min-width: 240px;
+					}
+
+					label + label {
+						padding: var(--ee-padding-nano) var(--ee-padding-smaller);
+					}
+
+					> div {
+						width: calc(100% - 240px);
+					}
+				}
+
+				@include max600px {
+					margin: var(--ee-margin-smaller) 0;
+				}
+
+				@include max480px {
+					margin: var(--ee-margin-smaller) 0;
+
+					label:first-child {
+						min-width: 200px;
+					}
+
+					> div {
+						width: calc(100% - 200px);
+					}
+				}
+
+				@include max360px {
+					margin: var(--ee-margin-tiny) 0;
+
+					&:not(.ee-form-item__switch) {
+						display: flex;
+						flex-flow: column;
+						justify-content: flex-start;
+						padding-right: 0;
+
+						label:first-child {
+							justify-content: flex-start;
+							width: 100%;
+						}
+
+						> div {
+							width: 100%;
+						}
+					}
+				}
 			}
-		}
-	}
 
-	@media only screen and (max-width: 782px) {
-		.section-heading {
-			flex: 0 100%;
-			h3 {
-				margin: 0 0 1.5rem;
+			// to override values set in &__pair
+			&__number > div {
+				width: 120px;
+
+				@include max360px {
+					width: 100%;
+				}
 			}
 
-			.section-heading-icon {
-				left: unset;
-				right: 0.25rem;
-				top: -0.75rem;
+			.ee-icon--info-circle-outlined {
```

Comment:
few weeks ago you asked me about some generated classnames in svgs, here you go, this is the reason;
I've added this one-time customization but should have better opted to control this via props